Src/PerformanceCollector/Filtering/AccumulatedValue.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Filtering
+{
+    using System.Collections.Concurrent;
+
+    /// <summary>
+    /// Accumulator for operationalized metrics.
+    /// </summary>
+    internal class AccumulatedValue
+    {
+        public readonly ConcurrentStack<double> Value = new ConcurrentStack<double>();
+
+        public AccumulatedValue(string metricId, AggregationType aggregationType)
+        {
+            this.MetricId = metricId;
+            this.AggregationType = aggregationType;
+        }
+
+        public string MetricId { get; }
```

Comment:
We could, but then it would need to be a field. I'd rather have a get-only property than a readonly public field, that feels weird :)